amundsen_application/static/js/ducks/bookmark/types.ts

```diff
@@ -14,7 +14,7 @@ export interface AddBookmarkRequest {
 }
 export interface AddBookmarkResponse {
   type: AddBookmark.SUCCESS | AddBookmark.FAILURE;
-  payload: {
+  payload?: {
```

Comment:
`?` could be removed if a default `[]` is returned for bookmark failures.

Reply:
Up for it depending on where we end up in regards to whether or not this needs to be included in the payload.